web/regression/re_sql/tests/test_resql.py

```diff
@@ -95,6 +95,35 @@ def setUp(self):
         # Added line break after scenario name
         print("")
 
+        # Create replication slot if it does not exist for the
```

Comment:
This can be created using precondition_sql, Check tests for extensions.
Something like - 
```
DO $$ BEGIN IF NOT EXISTS (...) THEN PERFORM pg_create_logical_replication_slot(...); END IF; END $$;
```
And use      
"post_scenario_sql": "DROP ROLE IF EXISTS test_default_priv_user;"